data/Music/Sonic the Hedgehog Series/Sonic Rush/VelaNova.meta

```diff
@@ -1,3 +1,4 @@
 Sonic Rush - Vela Nova
 0x03
-bgm
\ No newline at end of file
+bgm
+GanonBattle,GanondorfBattle
```

Comment:
The location line should have at least one group from each level of specificity (exact, high, mid and low).